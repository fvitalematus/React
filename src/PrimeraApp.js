import React from 'react';
// import React, { Fragment } from 'react'; 

// Funtional Components
const PrimeraApp = () => {

    const saludo = 'Hola Mundo';

    return (
        <>
            <h1> {saludo} </h1>
            {/*<pre> { JSON.stringify( saludo, null, 3 ) } </pre>*/}
            <p>Mi primera aplicaci√≥n</p>
        </>
    );
}

export default PrimeraApp;