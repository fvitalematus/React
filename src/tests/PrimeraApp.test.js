import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import PrimeraApp from '../PrimeraApp';


describe('Pruebas en <PrimeraApp />', () => {

    // test('debe mostrar el mensaje "Hola, Soy Tony"', () => {
    // const saludo = 'Hola, Soy Tony';
    // const {getByText} = render( <PrimeraApp saludo = { saludo } /> );
    // expect(getByText(saludo) ).toBeInTheDocument();
    // });

    test('debe mostrar <PrimeraApp /> correctamente ', () => {

        const saludo = 'Hola, Soy Tony';
        const wrapper = shallow(<PrimeraApp saludo={saludo} />)

        expect( wrapper ).toMatchSnapshot();

    });


})
