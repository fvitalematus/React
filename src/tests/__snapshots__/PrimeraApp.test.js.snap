// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> debe mostrar <PrimeraApp /> correctamente  1`] = `
<Fragment>
  <h1>
    Hola, Soy Tony
    !!!
  </h1>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
